src/shadow.rs

```diff
@@ -15,7 +15,7 @@ declare_lint!(pub SHADOW_REUSE, Allow,
     "rebinding a name to an expression that re-uses the original value, e.g. \
     `let x = x + 1`");
 declare_lint!(pub SHADOW_UNRELATED, Allow,
-    "The name is re-bound without even using the original value");
+     "The name is re-bound without even using the original value");
```

Comment:
Does that single space change anything about the code? Otherwise I think we may remove it from the change to reduce diff clutter. :smile: